nixos/modules/services/databases/pgmanage.nix

```diff
@@ -177,28 +176,41 @@ in {
     };
   };
 
+in {
+
+  # postage has been renamed to pgmanage. In order to get security updates we
+  # need to switch from postage to pgmanage in 17.09. However, just renaming the
+  # options from services.postage to services.pgmanage will cause evaluation
+  # failures for users which is not acceptable within a release branch. So we
+  # keep the existing services.postage:
+  options.services.postage = pgmanageOptions;
+  # And define the defaults of the new services.pgmanage in terms of services.postage:
+  options.services.pgmanage = flip mapAttrs pgmanageOptions (name: option:
+    option // { default = config.services.postage.${name}; }
+  );
+
   config = mkIf cfg.enable {
-    systemd.services.postage = {
-      description = "postage - PostgreSQL Administration for the web";
+    systemd.services.pgmanage = {
```

Comment:
not sure how important this is, but the systemd service will get renamed with these changes

Reply:
that one should be ok.

Reply:
Hrm, not if you have centralized logging or monitoring for that specific service.

Reply:
However that should not affect the functionality of the service itself. So a release note should OK.

Reply:
for 18.03 that's fine, but we should be more careful with changes on an active release IMHO

Reply:
ah, I did not notice that this was a backport.